pkgs/top-level/node-packages.nix

```diff
@@ -25,6 +25,15 @@ rec {
     "node-protobuf".buildInputs = [ pkgs.protobuf ];
     "rbytes".buildInputs = [ pkgs.openssl ];
 
+    # Remove fsevents dependency because it is OSX-specific.
```

Comment:
Ohh, that's prone to breaking, i should implement support for generating for platforms. But isn't fsevents part of optionalDependencies, and currently npm2nix does not even handle optionalDependencies.